linux/just_files/just_install_functions.bsh

```diff
@@ -627,6 +627,53 @@ function pipenv-install()
           ' >> "${pipenv_activate}"
 }
 
+#**
+# .. function:: vsbuild-install
+#
+# Install Visual Studio build tools.
+#
+# :Arguments: * [``--installer {INSTALLER}``] - VS installer (default= ``${VS_BUILD_TOOLS_INSTALLER:-}``)
+#             * [``--download``] - Download VS installer
+#
+# Specifying the installer is can be used for offline installations:
```

Comment:
Strange wording - are you trying to say "To specify an installer for offline installations"?